### PR TITLE
Rust latam

### PR DESCRIPTION
**Pull request by yoshuawuyts**

### CODE REVIEW

File: 2020-rust-latam.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# CFP Rust Latam
+
+- Title: Designing Tide & Surf
+- Tags: intermediate / advanced
+- Language: EN (though considering doing ES; native speaker, but never
+presented about tech in ES before -- might want to)
```
Comment (lrlna) [inline 2020-rust-latam.md]: i think if there is space to point out that you would want someone from the team to practice with you in Spanish, I think they will appreciate it.

File: 2020-rust-latam.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+## Abstract
+
+Rust is going through a meteoric rise - increasingly underpinning some of the
+largest infrastructure deployments in the world. Much of that has to do with
+networking, and is considered pretty advanced. But what if we told you that
+you too could be writing advanced networking code?
```
Comment (lrlna) [inline 2020-rust-latam.md:13]: i feel like this last line implies that people are not able to write advanced networking code. 

I think what you trying to do is say that rust's networking stuff can be convoluted, but you've made it less convoluted. Is that right?

File: 2020-rust-latam.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+## Committee Details
+
+The talk will start stating the importance for Rust to have a web presence,
+drawing on numbers from the Rust and StackOverflow surveys. We'll then share
+the work we've been doing to solve this: Surf & Tide, a framework pairing of
+HTTP server and HTTP client. After a brief introduction we'll cover several
+unique features they provide, how they've been made possible by Rust.
+
+The purpose of this talk is to lift the covers on the design process that
+goes into building something as complex as a web framework in Rust. Rust's is
```
Comment (lrlna) [inline 2020-rust-latam.md:29]: I really like this first sentence, I'd even add it to your general abstract. I think people really would want to learn design thinking, especially in rust, and that's what you want to give them, not just a talk introducing another framework.

File: 2020-rust-latam.md (hunk 1 above)
Comment (yoshuawuyts) [reply 2020-rust-latam.md]: That's a great suggestion, Thanks!

File: 2020-rust-latam.md (hunk 2 above)
Comment (yoshuawuyts) [reply 2020-rust-latam.md:13]: It's a very hard topic to break into, and people often confuse "performant" with "necessarily hard to use". I want to break out of that.

File: 2020-rust-latam.md (hunk 3 above)
Comment (yoshuawuyts) [reply 2020-rust-latam.md:29]: Ohhh, thanks! -- yeah I like your suggestion. Updating!

**Pull request merged**